omc3/model/accelerators/lhc.py

```diff
@@ -339,6 +339,10 @@ def get_exciter_bpm(self, plane: str, commonbpms: List[str]):
         return None
 
     def important_phase_advances(self) -> List[List[str]]:
+        if "hl" in self.year: 
+            # skip if HiLumi
```

Comment:
Would you add a TODO in this comment? Would make it easier to come back to when Riccardo and team are ready with this info